lib/public/components/Filters/common/filters/textFilter.js

```diff
@@ -16,12 +16,15 @@ import { h } from '/js/src/index.js';
 /**
  * Returns a text filter component
  *
- * @param {FilterInputModel} filterInputModel the model of the text filter
+ * @param {FilterInputModel|TextTokensFilterModel} filterInputModel the model of the text filter
  * @param {Object} attributes the additional attributes to pass to the component, such as id and classes
  * @return {Component} the filter component
  */
```

Comment:
I think that when you have such a case, you should wrap this in another component (like textTokensFilter)

Reply:
as I put here https://github.com/AliceO2Group/Bookkeeping/pull/1252#pullrequestreview-1752712398, I was going to entirely replace `FilterInputModel` by `TextTokensFilterModel`, so that's temporary